marklogic-data-hub/src/test/java/com/marklogic/hub/dataservices/ingestion/BulkIngestTest.java

```diff
@@ -0,0 +1,80 @@
+package com.marklogic.hub.dataservices.ingestion;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.dataservices.InputEndpoint;
+import com.marklogic.client.document.JSONDocumentManager;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.io.StringHandle;
+import com.marklogic.hub.ApplicationConfig;
+import com.marklogic.hub.HubTestBase;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.stream.Stream;
+
+import static org.junit.Assert.assertNotNull;
+
+@ExtendWith(SpringExtension.class)
+@ContextConfiguration(classes = ApplicationConfig.class)
+public class BulkIngestTest extends HubTestBase {
+
+    DatabaseClient db;
+
+    @BeforeEach
+    public void setupTest() {
+        db = adminHubConfig.newStagingClient(null);
+    }
+
+    @Test
+    public void testBulkIngest() {
+
+        String prefix = "/bulkIngestTest";
+        String endpointState = "{\"next\":" + 0 + ", \"prefix\":\""+prefix+"\"}";
+        String workUnit      = "{\"taskId\":"+1+"}";
+
+        InputEndpoint loadEndpt = InputEndpoint.on(adminHubConfig.newStagingClient(null),
+            adminHubConfig.newModulesDbClient().newTextDocumentManager().read("/data-hub/5/data-services/ingestion/bulkIngester.api", new StringHandle()));
+
+        InputEndpoint.BulkInputCaller loader = loadEndpt.bulkCaller();
+        loader.setEndpointState(new ByteArrayInputStream(endpointState.getBytes()));
+        loader.setWorkUnit(new ByteArrayInputStream(workUnit.getBytes()));
+
+        Stream<InputStream> input         = Stream.of(
+            asInputStream("{\"docNum\":1, \"docName\":\"doc1\"}"),
+            asInputStream("{\"docNum\":2, \"docName\":\"doc2\"}"),
+            asInputStream("{\"docNum\":3, \"docName\":\"doc3\"}")
+        );
+        input.forEach(loader::accept);
+        loader.awaitCompletion();
+        checkResults("/bulkIngestTest/1/1.json");
+        checkResults("/bulkIngestTest/1/2.json");
+        checkResults("/bulkIngestTest/1/3.json");
+    }
+
+    @AfterEach
+    public void cleanup(){
```

Comment:
Remove this - we actually want the data to stay after a test finishes. If the test fails, it makes debugging a lot easier. 

Instead of cleaning data up at the end of the test, the test plumbing in AbstractHubCoreTest will remove data at the start of each test.